abstracts/turing.Rmd

```diff
@@ -6,8 +6,13 @@ editor_options:
 
 # OpenInfra: Utilising OpenStreetMap for sustainable transport planning
 
-In a post pandemic -- high inflation world, active travel modes will become more important due to reduced public transport capacities and increased costs.
```

Comment:
There is still a pandemic! I've tried to re-word it to make it broader and more accessible.